client/tests/integration/domain_owner.rs

```diff
@@ -122,7 +122,7 @@ fn domain_owner_asset_definition_permissions() -> Result<()> {
 
     // register asset definitions by "bob@kingdom" so he is owner of it
     let coin = AssetDefinition::quantity(coin_id.clone());
-    let transaction = TransactionBuilder::new(bob_id.clone())
+    let transaction = TransactionBuilder::new(0, bob_id.clone())
```

Comment:
Maybe add `with_chain_id` with 0 by default? Because it's not really obvious what this 0 represent.